lib/ansible/modules/system/iptables.py

```diff
@@ -399,8 +399,14 @@ def construct_rule(params):
         False)
     append_match(rule, params['comment'], 'comment')
     append_param(rule, params['comment'], '--comment', False)
-    append_match(rule, params['ctstate'], 'state')
-    append_csv(rule, params['ctstate'], '--state')
+    if params['ctstate']:
```

Comment:
I just remember that it is not possible to use --state here, for "state" is reserved:
"msg": "value of state must be one of: present,absent, got: ESTABLISHED,RELATED"